### PR TITLE
Add admin log for codewords

### PR DESCRIPTION
**Pull request by SlamBamActionman**
## About the PR

Adds an admin log for codewords

## Why / Balance

Ret wanted it since the refactor removed it accidentally.

## Technical details

Logs when codewords are generated

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
ADMIN:
- add: Added admin logging for generated codewords.

### CODE REVIEW

File: Content.Server/GameTicking/Rules/TraitorRuleSystem.cs [hunk 1]
```diff
@@ -58,6 +61,7 @@ private void AfterEntitySelected(Entity<TraitorRuleComponent> ent, ref AfterAnta
     private void SetCodewords(TraitorRuleComponent component)
     {
         component.Codewords = GenerateTraitorCodewords(component);
+        _adminLogger.Add(LogType.EventStarted, LogImpact.Low, $"Codewords generated: {string.Join(", ", component.Codewords)}");
```
Comment (deltanedas) [inline Content.Server/GameTicking/Rules/TraitorRuleSystem.cs:64]: should log the rule too since multiple rules can have codewords

Comment (SlamBamActionman) [reply Content.Server/GameTicking/Rules/TraitorRuleSystem.cs]: Very good point, fixed it now

Just wanna say thanks for reviewing a small little admin PR like this btw, you're awesome

**Comment by SlamBamActionman** [comment]
image
Includes game rule entity now, to make it easy to connect the log to which game rule generated the codewords.

**Review by slarticodefast — APPROVED** [review]
Tested in-game and works as intended.

**Pull request merged**